python/testMP.py

```diff
@@ -9,19 +10,21 @@
     parser.add_argument("--port", default=5005, help="OSC server port")
     args = parser.parse_args()
 
-    r = rosary.Rosary(args.ip, args.port)
+    d = dispatcher.Dispatcher()
+    r = rosary.Rosary(args.ip, args.port, d)
 
     # create a bounce effect on all beads
-    r.add_effect('bounce', r.set_registry['All'])
+    #r.add_effect('bounce', r.set_registry['All'])
+    r.add_effect('bounce', 'all')
 
     # change color to red
-    r.effect(1).color.set(r.color_registry['Red'])
+    r.effect(1).color.set(r.color_registry['red'])
 
     # add another throb on the odd beads
-    r.add_effect('throb', r.set_registry['Odd_Ring'], color=r.color_registry['Blue'])
+    r.add_effect('throb', 'odd_ring', color_name_or_r='blue')
```

Comment:
I think we all agree by now that `color_name_or_r` was a terrible idea. I'll fix it later.